src/registry_schemas/schemas/consent_continuation_out.json

```diff
@@ -0,0 +1,29 @@
+{
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "$id": "https://bcrs.gov.bc.ca/.well_known/schemas/consent_continuation_out",
+    "title": "Consent Continuation Out Schema",
+    "definitions": {},
+    "required": [
+        "consentContinuationOut"
+    ],
+    "type": "object",
+    "properties": {
+        "consentContinuationOut": {
+            "type": "object",
+            "required": [
+                "orderDetails"
+            ],
+            "properties": {
+                "orderDetails": {
```

Comment:
Is this an _order_, or a note from the Registry?
It doesn't seem similar to a Registrar's Order, and it's not a _courtOrder_

Reply:
https://preview.uxpin.com/f3e9c0da57601f193d2f0c6b1946134b2ddeb2fe#/pages/160490823/comment/sitemap

Reply:
It's similar to Registrar's Order

Reply:
Created a generic schema called `order` and referred in registrars order, registrars notation and consent continuation out